docs/issue-to-earn.mdx

```diff
@@ -0,0 +1,228 @@
+
+
+### 2. Issue Credentials to User
+
+:::info[An "ID Card" is a Collection of Credentials]
+We define a digital ID card as a collection of credentials, issued by a brand to a user. We defined a credential as a piece of data about a user — like their phone number, address, or government ID.
+
+**To "issue an ID card" is just to issue credentials to a user for the first time.** (To "update an ID card" is just to issue subsequent credentials to that user.)
+:::
+
+**Call [`POST /credentials`](/api-overview#issue-credentials)** with a user identifier (a verified email or phone for the user) and one or more credentials. For each credential, include a `type` and `data` in accordance with one of our [data schemas](/schema).
+
+:::note[Ask Us for New Schemas If Needed]
+If we don't have a data schema that suits your needs, email us at Support@Verified.Inc) and we'll make one for you.
+:::
+
+For example, if you have a verified Social Security Number (SSN) and a full name for a user, you can issue them a `SsnCredential` and `FullNameCredential`. Just call [`POST /credentials`](/api-overview#issue-credentials) with the user's email or phone, the `SsnCredential` and `FullNameCredential` type, and the SSN and full name data:
+
+```typescript title="Example Issue Credentials Request Body"
+{
+  "email": "test@example.com",
+  "credentials": [
+    {
+      "type": "SsnCredential",
+      // based on the properties attribute of the schema
+      // highlight-start
+      // single object, not an array of objects, since Ssn is an atomic credential
+      "data": {
+        "ssn": "111223333"
+      },
+      // highlight-end
+      "expirationDate": 1893484800
+    },
+    {
+      "type": "FullNameCredential",
+      // based on the properties attribute of the schema
+      // highlight-start
+      // an array of objects, not a single object, since FullName is a composite credential
+      "data": [
+        {
+          "type": "FirstNameCredential",
+          "data": {
+            "firstName": "Richard"
+          }
+        },
+        {
+          "type": "LastNameCredential",
+          "data": {
+            "lastName": "Hendricks"
+          }
+        }
+      ]
+      // highlight-end
+    }
+  ]
+}
+```
+
+Format the credential data according to the `properties` attribute of the credential schema. In this case, the schema is for a `SsnCredential` which can found found at [`https://schema.verified.inc/jsonSchema/SsnCredential`](https://schema.verified.inc/jsonSchema/SsnCredential) and for a `FullNameCredential` which can be found at [`https://schema.verified.inc/jsonSchema/FullNameCredential`](https://schema.verified.inc/jsonSchema/FullNameCredential).
```

Comment:
I think this detailed info here ought to be removed. They can go to the schema docs page for this detailed info.